src/kademlia/routing_table.cpp

```diff
@@ -1100,7 +1100,7 @@ std::vector<node_entry> routing_table::find_node(node_id const& target
 		if (int(l.size()) > count)
 		{
 			// sort the nodes by how close they are to the target
-			std::sort(l.begin() + unsorted_start_idx, l.end()
+			std::nth_element(l.begin() + unsorted_start_idx, l.begin() + count, l.end()
```

Comment:
I think the comment should be changed to indicate that we don't need the nodes to be sorted, we just want the `count` closest nodes.

Reply:
I've updated the comments.